bin/process-test-multi

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/perl
+
+use Genome;
+use Revision;
+use Library;
+use Users;
+use JenkinsData;
+use Memoize qw(memoize);
+use Genome::Process::Test::IntegrationTests;
+
+use strict;
+use warnings;
+
+
+# flush output buffer after every write or print
+local $| = 1;
+
+log_environment();
+JenkinsData->validate_environment;
+
+# set the title of this process
+$0 = sprintf("%s %s # TEST_SPEC = %s", $^X, __FILE__, JenkinsData->test_spec);
+
+# Per Process Customizations
+print("Customizing test environment...\n");
+
+set_genome_software_result_test_name();
+
+# Done with Per Process Customizations
+print "\n";
+
+my $process = get_initial_process();
+wait_for_process($process);
+
+$process = UR::Context->current->reload('Genome::Process', id => $process->id);
```

Comment:
I think you can do `$process = UR::Context->current->reload($process)` or maybe even just `UR::Context->current->reload($process)`.